Character.cpp

```diff
@@ -86,8 +87,17 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
+void boosting(int& stat, int& initialVal)
+{
```

Comment:
`boosting` is not a good name.  

name things according to what they **do** or what they **are**.

This function "boosts the stats", so it should be given a similar name, such as `boostStats()`
- [ ] revise.